components/add_new_cv_form.tsx

```diff
@@ -97,26 +86,33 @@ export default function AddNewCvForm({ id }: Props) {
     }
 
     delete updatedCv.projects;
+    delete updatedCv.technical_skills;
 
     return supabase.from("cv").upsert(updatedCv).select();
   }
 
   async function upsertProjects(values: any, cvData: any) {
-    if (values.id) {
-      return supabase.from("projects").update(values).eq("id", values.id);
-    }
-    return supabase.from("projects").insert({ ...values, cv_id: cvData[0].id });
+    values.cv_id = cvData[0].id;
+    return supabase.from("projects").upsert(values);
+  }
+
+  async function upsertSkills(values: any, cvData: any) {
+    values.cv_id = cvData[0].id;
+    return supabase.from("technical_skills").upsert(values);
   }
 
   async function handleSubmit(values: any) {
     const { data, error } = await upsert(values);
-    values.projects.forEach((project: any) => upsertProjects(project, data));
-
-    if (error) {
-      setServerErrorMessage(error.message);
-    } else {
-      await router.push("/");
-    }
```

Comment:
Why did you remove the error handling for projects?

Reply:
because I already changed projects structure so upsertProject function here doesn't do anything, I commented that line now
i added server errors for projects in [this PR](https://github.com/bditengineering/cv-app/pull/16) with an updated upsertProjects function

Reply:
Understood. Why did you comment on the line instead of deleting it?

Reply:
no particular reason, deleted that line now instead